File: Heat/Models/Medium.vb
Namespace Models

    ''' <summary>
    ''' Rappresenta un oggetto di tipo grafico o audio o video o whatever caricata come allegato ad un'altra entità.
    ''' </summary>
    Public Class Medium

        Property ID As Integer
        Property OriginalFilename As String
        Property UploadFilename As String
        Property RelativePath As String
        Property AbsolutePath As String
        Property Lenght As Integer
        Property Extension As String
        Property ContentType As String
        Property Description As String
        Property Tags As String

    End Class
End Namespace

File: Heat.ConvertedToC#/Models/Medium.cs
namespace Heat.Models
{

    /// <summary>
    /// Rappresenta un oggetto di tipo grafico o audio o video o whatever caricata come allegato ad un'altra entità.
    /// </summary>
    public class Medium
	{

		public int ID { get; set; }
		public string OriginalFileName { get; set; }
		public string UploadFileName { get; set; }
		public string RelativePath { get; set; }
		public string AbsolutePath { get; set; }
		public int Lenght { get; set; }
		public string Extension { get; set; }
		public string ContentType { get; set; }
		public string Description { get; set; }
		public string Tags { get; set; }

	}
}

